moVirt/src/main/java/org/ovirt/mobile/movirt/ui/dashboard/maps/DcStatusMap.java

```diff
@@ -0,0 +1,42 @@
+package org.ovirt.mobile.movirt.ui.dashboard.maps;
+
+import org.ovirt.mobile.movirt.model.DataCenter;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by suomiy on 5/18/16.
+ */
+public enum DcStatusMap {
+    // dashboard position depends on the order
+    WARNING(DataCenter.Status.UNINITIALIZED, DataCenter.Status.MAINTENANCE, DataCenter.Status.CONTEND, DataCenter.Status.UNKNOWN),
```

Comment:
are you sure about this mappings? did you get them from the web version? because they don't seem too good to me... The non_operational looks as a warning while the maintenance looks as a normal down...